modules/gdpssp.R

```diff
@@ -40,7 +39,16 @@ annual_gdpg <- function(sd){
   return(list(year = g_years[2:(length(g_years))],gdpr = gdpg))
 }
 
+annual_gdp <- function(sd){
+  g_years <- seq(sd$year[1], sd$year[length(sd$year)], by = 1)
+  annual_gdp <- approx(sd$year, sd$gdp, g_years)$y
+  annual_gdp <- interp1(sd$year, sd$gdp, g_years, method = "spline")
```

Comment:
why do we have two techniques to calculate this? I prefer linear interpolation to splines as it avoids ringing problems.

Reply:
They used those methods as well to calculate annual_gdpr so I figured it would be best to stick as close as possible to their code. But I could change it.

Reply:
It's clearly a bug left in the code from trying several different techniques, comment out the upper line in both cases.